packages/OS400/os400sys.c

```diff
@@ -268,13 +268,9 @@ Curl_getnameinfo_a(const struct sockaddr * sa, curl_socklen_t salen,
               int flags)
 
 {
-  char * enodename;
-  char * eservname;
+  char * enodename = (char *)NULL;
+  char * eservname = (char *)NULL;
```

Comment:
this could also fix the code style: make it `char *ennodename` (without the extra space after the asterisk)

Reply:
Is this `(char *) `cast required for Codacy as we have `char *variable = NULL` elsewhere?

Reply:
@captain-caveman2k No it's not, I just moved the whole expression from below. I'll remove the cast.